Gemfile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Ignore frozen_string_literal in your rubocop.yml and remove this line in all files please